Asterix/optics/propagation_functions.py

```diff
@@ -585,20 +586,24 @@ def prop_fpm_regional_sampling(pup,
 
     AUTHOR: R. Galicher (in IDL)
             ILa (to Python)
+            J. Mazoyer : small modifs to adapt to Asterix Dec 22
 
     Parameters
     ----------
-    pup : 2D array
-        Input mage array containing the wavefront at the entrance pupil of the optical system.
+    pup : 2D array of size [self.dim_overpad_pupil,self.dim_overpad_pupil]
+        Input mage array containing the wavefront at the entrance pupil of the optical system. Be careful this si
```

Comment:
Something missing here

Reply:
I've actually corrected that in the other PR so let's ignore

Reply:
corrected